ui/src/authentication/pages/Login/Login.js

```diff
@@ -35,28 +47,181 @@ class LoginPage extends Component {
 
   render() {
     this.redirectIfAlreadyLoggedIn();
+
+    const notImplementedPopup = (
+      <Popup
+        content="Not implemented yet!"
+        trigger={<Link className="disabled">here</Link>}
+      />
+    );
+
     return (
-      <Grid
-        textAlign="center"
-        verticalAlign="middle"
-        columns={2}
-        style={{ height: '100vh', backgroundColor: '#f9f9f9' }}
-      >
-        <Grid.Column>
-          <Segment>
-            <Header as="h2" textAlign="center">
-              Login to account
-            </Header>
-            {ENABLE_OAUTH_LOGIN && <LoginWithOauthProviders />}
-            {ENABLE_LOCAL_ACCOUNT_LOGIN && (
-              <>
-                <Divider horizontal>Or</Divider>
-                <LoginWithLocalAccount />
-              </>
-            )}
-          </Segment>
-        </Grid.Column>
-      </Grid>
+      <div className="frontsite">
+        <Container
+          fluid
+          className="auth-page blur"
+          style={{
+            backgroundImage: this.props.backgroundImage
+              ? `url(${this.props.backgroundImage})`
+              : null,
+          }}
+        >
+          <Container>
+            <Segment
+              className="background-transparent pb-default pt-default"
+              color="orange"
+            >
+              <Grid
+                stretched
+                verticalAlign="middle"
+                textAlign="center"
+                centered
+                columns={2}
+                padded
+              >
+                <Grid.Row only="mobile">
+                  <Grid.Column width={16}>
+                    <Image
+                      src={
+                        'https://inveniosoftware.org/static/img/logo-invenio-ils.svg?h=da7ca04b'
```

Comment:
I would add the logo in the repo :)

Reply:
it was temporary, until you choose the variant :)
but on the other hand, it would be nice to have a resource for logo, we change it in one place and then it changes in all of the websites...